projects/js-packages/publicize-components/src/hooks/use-share-post/index.js

```diff
@@ -82,7 +82,7 @@ export default function useSharePost( postId ) {
 	const path = getSocialScriptData().api_paths.resharePost.replace( '{postId}', postId );
 
 	const doPublicize = useCallback(
-		function () {
+		function ( overrideSkippedConnections = null ) {
```

Comment:
Is `overrideSkippedConnections` the best name here? 

I suppose the intent is to not share to anything passed in `overrideSkippedConnections`. Is that right?

Reply:
Yeah, we can name it something like `connectionsToSkip` and it should be implicit that it overrides that existing list.

Reply:
I can rename that, I used this name because we already have skipped connections from a hook above, and this will overwrite that value if passed

Reply:
Done in [38ba700](https://github.com/Automattic/jetpack/pull/39157/commits/38ba7002576e0a40c588b66ae764aa03258f2de2)